src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How The Merge impacts ether supply
+description: Breakdown on how The Merge will impact ether supply
+lang: en
+sidebar: true
+---
+
+# How The Merge impacts ether supply {#how-the-merge-impacts-ether-supply}
+
+<Card
+emoji=":chart_decreasing:">
+
+- Mining rewards ~13,000 ETH/day pre-merge
+- Staking rewards ~1,600 ETH/day pre-merge
+- **After The Merge, only the ~1,600 ETH per day will remain, dropping total new ETH issuance by ~90%**
+- <Emoji text=":fire:" size="1" /> The burn: At an average gas price of at least 16 gwei, at least 1,600 ETH is burned every day, which effectively brings net ETH inflation to zero or less post-merge.
+
+</Card>
+
+We can break the supply of ETH into two primary forces: issuance, and burn.
+
+The **issuance** of ETH is the process of creating ETH that did not previously exist on the balance sheet, while the **burning** of ETH refers to when existing ETH is permanently removed from circulation. The rate of each of these fluctuates based on a number of parameters, and the balance between them determines the resulting inflation/deflation rate of ether.
+
+The way ETH is issued will undergo some changes at time of The Merge. Currently, new ETH is issued from two main sources: the execution layer (EL, i.e. Mainnet) and the consensus layer (CL, i.e. Beacon Chain). After The Merge, issuance from the execution layer will go to zero. Let's break this down.
+
+## Pre-merge {#pre-merge}
+
+### EL Issuance {#el-issuance-pre-merge}
```

Comment:
I feel pretty strongly that we should spell out execution layer in full. Otherwise, it'll be a headache for translation. Also, two-letter acronyms are relatively uncommon and in this specific case they'll always add cognitive complexity IMO.

Reply:
Commented elsewhere, but my view is that acronyms are a layer of abstraction. Some research has shown that recently learned acronyms take more mental processing than fully spelt-out terms despite being shorter in length. 

Re: style guide, yeah, it's not ideal. Personally, I'm fine with the more common acronyms (NFTs, dapps, DeFi, DAOs) but when it starts to get less well-known I struggle to justify their use for beginners espescially.